content/tutorials/portenta-h7/por-ard-trace32/content.md

```diff
@@ -158,6 +162,22 @@ In this tutorial you learned how to acquire a free version of the TRACE32 GDB Fr
 
 # Troubleshooting
 
+### Portenta's serial number is not 24 digit long:
+
+* Update Arduino IDE to the latest version available
+* Update **Arduino mbed-enabled Boards** core from Arduino IDE menu: _Tools > Board > Boards Manager_
+* Update the Portenta's bootloader [here](https://www.arduino.cc/pro/tutorials/portenta-h7/por-ard-bl).
+
+
+
+### Error: 'No more arguments expected' 
+
+* This may be caused by your Windows user name. Check there is no 'spaces' or special characters within the user name to avoid the error.
```

Comment:
```suggestion
- This may be caused by unsupported characters in your Windows user name. Make sure your user name neither contains any spaces nor special characters.
```